ui/components/app/detected-token/detected-token-values/detected-token-values.js

```diff
@@ -13,32 +13,47 @@ import {
 import { useTokenTracker } from '../../../../hooks/useTokenTracker';
 import { useTokenFiatAmount } from '../../../../hooks/useTokenFiatAmount';
 
-const DetectedTokenValues = ({ token }) => {
-  const [selectedTokens, setSelectedTokens] = useState(false);
+const DetectedTokenValues = ({
+  token,
+  handleTokenSelection,
+  selectedTokens,
+}) => {
+  const [selectToken, setSelectToken] = useState(() => {
+    return Boolean(
+      selectedTokens.filter((address) => token.address === address),
+    );
+  });
```

Comment:
This will always be true, because if it finds no match it'll return an empty array which Boolean([]) is true

```
Boolean([].filter(x => x === 1))
> true
```